### PR TITLE
Feature: Google Keep as Todos

### LINKED ISSUE #197: Feature: Google Keep as Todos
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

Although the Todos currently supports Google Tasks I'm sometimes using Google Keep although it's not the best approach for managing task, is it possible to add that as well?

### Proposed Solution

Adding Google Keep as a Todos option.

### Alternatives Considered

Google Tasks, Microsoft ToDo, I don't like any of these.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by SpecialAro**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change
Feature: Google Keep as Todos

#### Motivation and Context
Feature: Google Keep as Todos. Fix #197, even though we have an option of setting a custom TODO URL as server.

#### Screenshots
| image | image |
|-- |-- |

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -110,6 +111,8 @@ export const TODO_SERVICE_RECIPE_IDS = {
   [TODO_MSTODO_URL]: 'mstodo',
   [TODO_HABITICA_URL]: 'habitica',
   [TODO_ANYDO_URL]: 'anydo',
+  [TODO_GOOGLETASKS_URL]: 'Google Tasks',
```
Comment (vraravam) [inline src/config.ts]: these need to refer to the names of recipes - there's no recipe called "Google Tasks"

File: src/config.ts [hunk 2]
```diff
@@ -110,6 +111,8 @@ export const TODO_SERVICE_RECIPE_IDS = {
...
+  [TODO_GOOGLEKEEP_URL]: 'google-keep',
```
Comment (vraravam) [inline src/config.ts]: the recipe name is "googlekeep"

**Review by vraravam — CHANGES_REQUESTED** [review]

**Review by vraravam — APPROVED** [review]

**Comment by yarons** [comment]
I'm wondering if there's a way to make it look good in a narrow sidebar, there are lots of missing details in narrow screen.

**Comment by SpecialAro** [comment]
> I'm wondering if there's a way to make it look good in a narrow sidebar, there are lots of missing details in narrow screen.

Most likely by injecting some CSS code you can achieve it.

**Pull request merged**